core/src/main/java/org/apache/calcite/sql/dialect/BigQuerySqlDialect.java

```diff
@@ -702,6 +703,31 @@ private void unparseOtherFunction(SqlWriter writer, SqlCall call, int leftPrec,
       call.operand(1).unparse(writer, leftPrec, rightPrec);
       writer.endFunCall(frame);
       break;
+    case "TIMESTAMPINTMUL":
+      if (call.getOperandList().get(0).getClass().getName()
+              .equals("org.apache.calcite.sql.SqlBasicCall")) {
```

Comment:
check call.operand(0) instance of SqlBasicCall. inspite of checking with className